src/pyop/storage.py

```diff
@@ -120,7 +120,7 @@ class RedisWrapper(StorageBase):
     Supports JSON-serializable data types.
     """
 
-    def __init__(self, db_uri, collection, ttl=None, options=None):
+    def __init__(self, db_uri, collection, ttl=None, options={}):
```

Comment:
that way we can recover if users set options to `None` explicitly

```suggestion
    def __init__(self, db_uri, collection, ttl=None, options={}):
        if options is None:
            options = {}
```

Reply:
hm, I think we can also do without this suggestion.